escheduler-server/src/main/java/cn/escheduler/server/worker/task/http/HttpTask.java

```diff
@@ -0,0 +1,310 @@
+                    exitStatusCode = -1;
+                }
+                break;
+            default:
+                if (!"200".equals(statusCode)) {
+                    appendMessage(httpParameters.getUrl() + " statuscode: " + statusCode + ", Must be: 200");
+                    exitStatusCode = -1;
+                }
+                break;
+        }
+        return exitStatusCode;
+    }
+
+    public String getOutput() {
+        return output;
+    }
+
+    protected void appendMessage(String message) {
+        if (output == null) {
+            output = "";
+        }
+        if (message != null && !message.trim().isEmpty()) {
+            output += message;
+        }
+    }
+
+    protected void addRequestParams(RequestBuilder builder,Map<String, Property> paramsMap) {
+        Iterator<Map.Entry<String, Property>> iter = paramsMap.entrySet().iterator();
+        while (iter.hasNext()){
+            Map.Entry<String, Property> en = iter.next();
+            Property property = en.getValue();
+            if (property.getValue() != null && property.getValue().length() > 0){
+                if (property.getDirect().equals(Direct.IN)){
+                    builder.addParameter(property.getProp(), property.getValue());
+                }else{
+                    if(CONTENT_TYPE.equals(property.getProp()) && property.getValue().contains(APPLICATION_JSON)){
+                        StringEntity postingString = null;
+                        try {
+                            postingString = new StringEntity("{}");
+                        } catch (UnsupportedEncodingException e) {
+                            e.printStackTrace();
+                        }
+                        builder.setEntity(postingString);
+                    }
+                }
+            }
+        }
+    }
+
+    protected void setHeaders(HttpUriRequest request,Map<String, Property> paramsMap) {
+        Iterator<Map.Entry<String, Property>> iter = paramsMap.entrySet().iterator();
+        while (iter.hasNext()){
+            Map.Entry<String, Property> en = iter.next();
+            Property property = en.getValue();
+            if (property.getValue() != null && property.getValue().length() > 0){
+                if (property.getDirect().equals(Direct.OUT)){
+                    request.addHeader(property.getProp(), property.getValue());
+                }
+            }
+        }
+    }
+
+    protected CloseableHttpClient createHttpClient() {
+        final RequestConfig requestConfig = requestConfig();
+        HttpClientBuilder httpClientBuilder;
+        httpClientBuilder = HttpClients.custom().setDefaultRequestConfig(requestConfig);
+        return httpClientBuilder.build();
+    }
+
+    private RequestConfig requestConfig() {
+        return RequestConfig.custom().setSocketTimeout(MAX_CONNECTION_MILLISECONDS).setConnectTimeout(MAX_CONNECTION_MILLISECONDS).build();
+    }
+
+    private SSLContext createSSLContext() {
```

Comment:
This function is not used anymore, please delete this function, can HttpsTrustManager ,   AND delete unused imports from this file.